index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav class="navbar" id="nav-bar">     
+      <img src="images/product-landing-page-logo.png" alt="Header Image" id="header-img">
+      <ul class="nav-link">
+        <li><a href="#features">Features</a></li>
+        <li><a href="#how-it-works">How it works</a></li>
+        <li><a href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>
+    <section class="heading" id="header">
+      <h1>Handcrafted, home-made masterpieces</h1>
+    </section>
```

Comment:
- [ ] Kindly explain why the header tag is better than the section tag in that area.